images/windows/scripts/helpers/VisualStudioHelpers.ps1

```diff
@@ -45,7 +41,7 @@ Function Install-VisualStudio {
     $channelUri = "https://aka.ms/vs/${Version}/${Channel}/channel"
     $channelId = "VisualStudio.${Version}.Release"
     $productId = "Microsoft.VisualStudio.Product.${Edition}"
-    $installChannelUri = "https://aka.ms/vs/${Version}/${Channel}/${installchannel}/channel"
+    $installChannelUri = "https://aka.ms/vs/${Version}/${Channel}/channel" 
```

Comment:
`installChannelUri` is the same as `channelUri`. Curious, do we still need providing it in the request? And if so, then you might consider changing this line to `$installChannelUri = $channelUri` for better visibility